addons/project/models/project.py

```diff
@@ -945,6 +948,11 @@ def _compute_subtask_planned_hours(self):
         for task in self:
             task.subtask_planned_hours = sum(child_task.planned_hours + child_task.subtask_planned_hours for child_task in task.child_ids)
 
+    @api.depends('child_ids')
+    def _compute_child_text(self):
+        for task in self:
+            task.child_text = task.subtask_count > 0 and _("(+ %(child_count)s task%(plural)s)", child_count=task.subtask_count, plural="s" if task.subtask_count > 1 else "") or False
```

Comment:
I was a bit lost, something like this is more readable what do you think ? 
```suggestion
            if not task.subtask_count:
                task.child_text = False
            elif task.subtask_count == 1:
                task.child_text = _("(+ 1 task)")
            else:
                task.child_text = _("(+ %s tasks)", task.subtask_count)
```